src/main/java/com/alkemy/ong/service/impl/NewsServiceImpl.java

```diff
@@ -18,10 +21,23 @@ public class NewsServiceImpl implements INewsService {
     @Autowired
     private NewsMapper newsMapper;
 
+    @Autowired
+    private  MessageSource message;
+
 
     @Override
     public NewsDto getById(Long id) {
         News news = newsRepository.getById(id);
         return newsMapper.entityToDto(news);
     }
+
+    public void deleteNews(Long id){
+
+        if (!newsRepository.existsById(id)){
+            throw new NullModelException(message.getMessage("error.null_model", null, Locale.US));
```

Comment:
Esto esta perfecto, pero hay un detalle, el NullModelException esta devolvindo un BAD REQUEST, y si no encuentra el elemento el codigo de error correcto seria un NOT FOUND.
Aca como esta implementado el NullModelExceptino
```
@ExceptionHandler(NullListException.class)
    public ResponseEntity<ApiErrorResponse<String>> handleDataRepresentationException(NullListException ex) {
        return new ResponseEntity<>(
                new ApiErrorResponse<>(
                        HttpStatus.BAD_REQUEST,
                        ex.getMessage()
                ),
                HttpStatus.BAD_REQUEST
        );
    }
```
Entonces, o se cambia la devolucion de esa implementacion o se realiza una diferente, todo segun como se este utilizando esto en los demas casos, pero siempre quie busque algo y no lo encuentre debo arrojar un NOT FOUND

Reply:
Ya lo arregle turco , si se hizo quilombo con los tickets, ya lo tengo en cuenta para la proxima